specification/elasticsan/resource-manager/Microsoft.ElasticSan/preview/2024-07-01-preview/examples/ElasticSans_Create_MaximumSet_Gen.json

```diff
@@ -0,0 +1,107 @@
+{
+  "title": "ElasticSans_Create",
+  "operationId": "ElasticSans_Create",
+  "parameters": {
```

Comment:
Compared to the examples of api version 2024-06-01-preview, this new sample seems to be missing the auto scale policy parameters. Same for update elasticSan example

Reply:
Updated